projects/ids-enterprise-ng/src/lib/field-filter/soho-field-filter.d.ts

```diff
@@ -44,18 +44,21 @@ interface JQuery<TElement = HTMLElement> extends Iterable<TElement> {
   fieldfilter(settings?: SohoFieldFilterSettings): JQuery;
 }
 
+type SohoFieldFilterOperator =
+  'end-with' | 'does-not-end-with' |
+ 'start-with' | 'does-not-start-with' |
+ 'equals' | 'does-not-equal' |
+ 'contains' | 'does-not-contain' |
+ 'calendar' | 'in-range' |
+ 'is-empty' | 'is-not-empty' |
+ 'less-equals' | 'less-than' |
+ 'greater-equals' | 'greater-than' |
+ 'between' | 'selected-notselected' |
+ 'selected' | 'not-selected' |
+ 'sort-a-to-z' | 'sort-z-to-a';
```

Comment:
Btw just found out, this looks identical to SohoDataGridColumnFilterConditions in soho-datagrid.d.ts. However that one lacks some of the values, I wonder which one is more correct?
Here's their difference: (left: SohoFieldFilterOperator, right: SohoDataGridColumnFilterConditions )
![image](https://user-images.githubusercontent.com/11013464/82274711-6f140480-99b3-11ea-84b3-30025acc78d2.png)